templates/controllers/grid/navigationMenus/categoriesNMIType.tpl

```diff
@@ -0,0 +1,17 @@
+{**
+ * templates/controllers/grid/navigationMenus/categoriesNMIType.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Custom categories NMI Type edit form part
+ *}
+{fbvFormSection id="NMI_TYPE_CATEGORY" class="NMI_TYPE_CUSTOM_EDIT" title="manager.navigationMenus.form.navigationMenuItem.category" for="categorySelect"}
```

Comment:
In the templates we can reference a PHP constant with `$smarty.const.NMI_TYPE_CATEGORY``. So for each of the templates, maybe use:

```
{fbvFormSection id=$smarty.const.NMI_TYPE_CATEGORY ...
```